addons/IWB_Toolbar/src/presenter.js

```diff
@@ -1345,6 +1349,14 @@ function AddonIWB_Toolbar_create() {
         });
     };
 
+    function isJQueryObject(element) {
+        return !!element[0];
+    }
+
+    function isFloatingImageBtnVisible() {
+        return $('.floating-image').css('display') === 'block';
```

Comment:
Ze względu, że w Edit CSS można ustawiać style to powinno być `return $('.floating-image').css('display') !== 'none';`
To na wypadek, gdyby klient ustawił wartość. np. inline